Cognite.Simulator.Utils/ScopedRemoteApiSink.cs

```diff
@@ -0,0 +1,92 @@
+using Serilog.Core;
+using Serilog.Events;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Cognite.Simulator.Extensions;
+using Cognite.Extractor.Utils;
+using CogniteSdk.Alpha;
+
+namespace Cognite.Simulator.Utils {
+
+    /// <summary>
+    /// Represents a sink for emitting log events to a remote API.
+    /// </summary>
+    /// 
+    public class ScopedRemoteApiSink : ILogEventSink
+    {
+        private readonly CogniteDestination cdfClient;
```

Comment:
can we somehow receive cognite client by leveraging the dependency injection? 
@lprovensi ?

Reply:
🤔 
I don't think so because the sink is currently built manually, instead of having the dependencies resolved by the provider. We would have to experiment a bit with the code injecting the logger to try it out.